src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -55,6 +58,7 @@ protected override ILspServices ConstructLspServices()
         }
 
         services.AddSingleton<ILspLogger>(_logger);
+        services.AddSingleton<ILogger>((LoggerWrapper)_logger);
```

Comment:
Consider something a bit less implementation specific that would work in other scenarios. For example:

```suggestion
        if (_logger is ILogger logger)
        {
            services.AddSingleton<ILogger>(logger);
        }
```